### PR TITLE
TELCODOCS-1957: Hardware accelerator overview

### PR DESCRIPTION
**Pull request by StephenJamesSmith**
[TELCODOCS-1957]: Hardware accelerator overview

New main topic. This Assembly will contain included subtopics for hardware accelerators such as NVIDIA, AMD, Intel, and others as they are developed. 

Jira: https://issues.redhat.com/browse/TELCODOCS-1957

Version(s):  openshift-4.17, openshift-4.16.z 

Link to docs preview: https://82500--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hardware_accelerators/about-hardware-accelerators.html

SME: @egallen 
QE: @bthurber

### CODE REVIEW

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+toc::[]
+
+[IMPORTANT]
+====
+Hardware accelerators are a Technology Preview feature only. Technology Preview features are not supported with Red{nbsp}Hat production service level agreements (SLAs) and might not be functionally complete. Red{nbsp}Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red{nbsp}Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+====
+
```
Comment (bthurber) [inline hardware_accelerators/about-hardware-accelerators.adoc:15]: Is there a reason we are calling this tech preview?  All of our accelerator tools like KMM, NFD and DTK are fully supported.  @egallen thoughts?

**Review by bthurber — COMMENTED** [review]
Question on tech preview status.

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+Hardware Operators use the operating framework of a Kubernetes cluster to enable the required accelerator resources. A device plugin is also provided that can be deployed manually or as a DaemonSet. This plugin enables the registration of the GPU in the cluster.   
+
+== Supported vendors
+
+Red{nbsp}Hat Openshift supports hardware accelerators provided by the following vendors:
+
+* AMD
+* NVIDIA
+* Intel
+
```
Comment (bthurber) [inline hardware_accelerators/about-hardware-accelerators.adoc:44]: Do we support these for OpenShift?  I don't think so.  We provide accelerator tools.  The partners are responsible for their operators which integrate with our accelerator tools.  

RHEL AI is a different topic.  In this case Red Hat does support the partner accelerator specific images and content.

@egallen thoughts?

**Review by bthurber — COMMENTED** [review]
I believe some areas are too vague and we need to clarify.  Added comments.

**Comment by StephenJamesSmith** [comment]
@bthurber @egallen Please provide a review.

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+:context: gpu-accelerators
+
+toc::[]
+
+Specialized hardware accelerators play a key role in the emerging generative artificial intelligence and machine learning (AI/ML) industry. Specifically, hardware accelerators are essential to the training and serving of large language and other foundational models that power this new technology. It allows data scientists, data engineers, ML engineers, and developers to take advantage of the specialized hardware acceleration for data-intensive transformations and model development and serving. Much of that ecosystem is open source with a number of contributing vendors and open source foundations.
+
+Red{nbsp}Hat OpenShift provides support for cards and peripheral hardware that add processing units that comprise hardware accelerators:
+
+* Graphical processing units (GPUs)
+* Tensor processing units (TPUs)
```
Comment (egallen) [inline hardware_accelerators/about-hardware-accelerators.adoc:14]: Could we remove this TPU line?

**Review by egallen — COMMENTED** [review]
If we remove the TPU line, /LGTM

**Comment by StephenJamesSmith** [comment]
@egallen @bthurber Latest changes added. Still waiting for graphic.

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+* Hybrid-cloud support. On-premise support for model development, delivery, and deployment. 
+* Support for AI/ML workloads. Model testing, iteration, integration, promotion, and serving into production as services.
+
+Red{nbsp}Hat provides an optimized platform to enable these specialized hardware accelerators in our RHEL and OpenShift platforms at the Linux (kernel and userspace) and Kubernetes layers. To do this, Red{nbsp}Hat combines the proven capabilities of Red{nbsp}Hat OpenShift AI and Red{nbsp}Hat OpenShift in a single enterprise-ready AI application platform. 
+
+image::OCP_HW_Accelerators_2.png[OCP hardware accelerators]
+
+Red{nbsp}Hat OpenShift AI is an open hybrid AI/ML platform built on Red{nbsp}Hat OpenShift that enables enterprises to create and deliver AI-enabled applications securely and at scale across hybrid clouds. It enables data acquisition and preparation, model training and fine-tuning, model serving and model monitoring, and hardware acceleration. 
+
+Specialized hardware accelerators are designed to work within air gapped (disconnected) environments. This is because of customer requirements to maintain a secure environment for development and testing.
```
Comment (egallen) [inline hardware_accelerators/about-hardware-accelerators.adoc:30]: Could you change the sentence to "Certain specialized hardware accelerators are designed to work within air gapped (disconnected) environments."

File: images/OCP_HW_Accelerators_2.png
Comment (bthurber) [inline images/OCP_HW_Accelerators_2.png:1]: What' the third row supposed to represent?  Having the columns on-top of each other seems a bit confusing.  For example, GPU's only being physical and NPU's ony being virtual.  I vote to remove the 3rd row.

**Comment by StephenJamesSmith** [comment]
@egallen New graphic added. Please review.

**Comment by StephenJamesSmith** [comment]
@egallen @bthurber Latest changes added. Please review!

**Review by bthurber — APPROVED** [review]
/lgtm

**Comment by StephenJamesSmith** [comment]
@soflaher Please review and vet for terminology issues. Thx!

**Comment by soflaher** [comment]
> @soflaher Please review and vet for terminology issues. Thx!

Ack, am following up with Rob and Erwan and will get back to you as soon as we agree on the most accurate description.

**Comment by soflaher** [comment]
@StephenJamesSmith response from Rob:

> I'm not certain I have the full context, however I can share that I would avoid the validated accelerators statement.  ...also I would guide to avoid the word vendor when we are talking about a partner.
> If the company in question is a partner we should use the word partner i.e. nvidia, amd, intel qualcomm, amazon, ibm are all partners. 
> I can also share that OpenShift AI does not itself include any support for the accelerator hardware.  the support for the accelerator hardware comes from the partners in their own software and so they have to validate that their software&hardware are achieveing the desired behaviors.
> 
> In RHEL AI, Red Hat is using the partners as a supplier and including their bits in our software product. I have not seen how support actually plays out yet in RHEL AI. Also in RHEL AI, today the hardware systems are published as vendor validated
> 
> We are working on moving towards a certification but there are several gaps, which are being addressed

Given Rob's input and @bthurber 's above, some considerations for terminology:

- OpenShift can't follow terminology that OpenShift AI is doing 
- Rob recommends using "partner" where someone is a partner, instead of vendor.
- We need to be clear that the partner owns their part of the validation of their stack where our stuff is being used (or at least ensure we aren't explicitly stating support where the onus is on partners)
- Would  "partner validated" possibly work?:  Red Hat OpenShift provides accelerator tools that enable the following partner-validated hardware accelerators (or maybe "[RH OCP] provides accelerator tools that the following partners validate with their hardware accelerators"?

File: hardware_accelerators/about-hardware-accelerators.adoc (hunk 2 above)
Comment (egallen) [reply hardware_accelerators/about-hardware-accelerators.adoc:37]: Yes, better to remove the word support.

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+Red{nbsp}Hat provides an optimized platform to enable these specialized hardware accelerators in our RHEL and OpenShift platforms at the Linux (kernel and userspace) and Kubernetes layers. To do this, Red{nbsp}Hat combines the proven capabilities of Red{nbsp}Hat OpenShift AI and Red{nbsp}Hat OpenShift in a single enterprise-ready AI application platform. 
+
+Hardware Operators use the operating framework of a Kubernetes cluster to enable the required accelerator resources. A device plugin is also provided that can be deployed manually or as a DaemonSet. This plugin enables the registration of the GPU in the cluster. 
+
+Certain specialized hardware accelerators are designed to work within air gapped (disconnected) environments. This is because of customer requirements to maintain a secure environment for development and testing.  
+
+== Supported AI accelerators
+
+Red{nbsp}Hat Openshift supports the following AI accelerators:
```
Comment (egallen) [inline hardware_accelerators/about-hardware-accelerators.adoc:36]: we can replace "supports" by "enables"

**Review by egallen — COMMENTED** [review]
/lgtm

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 6]
```diff
@@ -0,0 +1,61 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="about-hardware-accelerators"]
+= About hardware accelerators
+include::_attributes/common-attributes.adoc[]
+:context: gpu-accelerators
```
Comment (maxwelldb) [inline hardware_accelerators/about-hardware-accelerators.adoc:5]: This should be the same as the anchor ID: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+toc::[]
+
+Specialized hardware accelerators play a key role in the emerging generative artificial intelligence and machine learning (AI/ML) industry. Specifically, hardware accelerators are essential to the training and serving of large language and other foundational models that power this new technology. It allows data scientists, data engineers, ML engineers, and developers to take advantage of the specialized hardware acceleration for data-intensive transformations and model development and serving. Much of that ecosystem is open source with a number of contributing partners and open source foundations.
+
+Red{nbsp}Hat OpenShift provides support for cards and peripheral hardware that add processing units that comprise hardware accelerators:
```
Comment (maxwelldb) [inline hardware_accelerators/about-hardware-accelerators.adoc:11]: Remember to use the product title attribute rather than "OpenShift" in most cases.

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+* Graphical processing units (GPUs)
+* Neural processing units (NPUs)
+* Application-specific integrated circuits (ASICs)
+* Data processing units (DPUs)
+* Others
+
+image::OCP_HW_Accelerators_4.png[OCP hardware accelerators]
```
Comment (maxwelldb) [inline hardware_accelerators/about-hardware-accelerators.adoc:19]: Alt text needs some fleshing out to be equivalent to the image.

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+Hybrid-cloud support:: On-premise support for model development, delivery, and deployment. 
+Support for AI/ML workloads:: Model testing, iteration, integration, promotion, and serving into production as services.
+
+Red{nbsp}Hat provides an optimized platform to enable these specialized hardware accelerators in our RHEL and OpenShift platforms at the Linux (kernel and userspace) and Kubernetes layers. To do this, Red{nbsp}Hat combines the proven capabilities of Red{nbsp}Hat OpenShift AI and Red{nbsp}Hat OpenShift in a single enterprise-ready AI application platform. 
+
+Hardware Operators use the operating framework of a Kubernetes cluster to enable the required accelerator resources. A device plugin is also provided that can be deployed manually or as a DaemonSet. This plugin enables the registration of the GPU in the cluster. 
+
+Certain specialized hardware accelerators are designed to work within air gapped (disconnected) environments. This is because of customer requirements to maintain a secure environment for development and testing.  
+
+== Hardware accelerators
```
Comment (maxwelldb) [inline hardware_accelerators/about-hardware-accelerators.adoc:34]: Missing anchor ID.

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+
+Red{nbsp}Hat Openshift enables the following hardware accelerators:
+
+* NVIDIA GPU
+* AMD Instinct GPU
+* Intel Gaudi
+
+.Prerequisites
```
Comment (maxwelldb) [inline hardware_accelerators/about-hardware-accelerators.adoc:42]: Prerequisites in this style are used for procedure modules. Is there a task that should follow here?

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 11]
```diff
@@ -0,0 +1,61 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="about-hardware-accelerators"]
+= About hardware accelerators
+include::_attributes/common-attributes.adoc[]
+:context: gpu-accelerators
+
+toc::[]
+
+Specialized hardware accelerators play a key role in the emerging generative artificial intelligence and machine learning (AI/ML) industry. Specifically, hardware accelerators are essential to the training and serving of large language and other foundational models that power this new technology. It allows data scientists, data engineers, ML engineers, and developers to take advantage of the specialized hardware acceleration for data-intensive transformations and model development and serving. Much of that ecosystem is open source with a number of contributing partners and open source foundations.
```
Comment (maxwelldb) [inline hardware_accelerators/about-hardware-accelerators.adoc:9]: This assembly should be [modularized](https://redhat-documentation.github.io/modular-docs/). At the moment, it looks like a concept module followed by a reference module to me.

**Review by maxwelldb — COMMENTED** [review]
Left some comments and suggestions. Re-label when ready. Thanks!

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 12]
```diff
@@ -0,0 +1,53 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="about-hardware-accelerators"]
+= About hardware accelerators
+include::_attributes/common-attributes.adoc[]
+:context: about-hardware-accelerators
+
+toc::[]
+
+Specialized hardware accelerators play a key role in the emerging generative artificial intelligence and machine learning (AI/ML) industry. Specifically, hardware accelerators are essential to the training and serving of large language and other foundational models that power this new technology. It allows data scientists, data engineers, ML engineers, and developers to take advantage of the specialized hardware acceleration for data-intensive transformations and model development and serving. Much of that ecosystem is open source with a number of contributing partners and open source foundations.
```
Comment (lahinson) [inline hardware_accelerators/about-hardware-accelerators.adoc:9]:
```suggestion
Specialized hardware accelerators play a key role in the emerging generative artificial intelligence and machine learning (AI/ML) industry. Specifically, hardware accelerators are essential to the training and serving of large language and other foundational models that power this new technology. Data scientists, data engineers, ML engineers, and developers can take advantage of the specialized hardware acceleration for data-intensive transformations and model development and serving. Much of that ecosystem is open source, with a number of contributing partners and open source foundations.
```
Is (AI/ML) an approved abbreviation? I see AI listed as approved in [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#AI), but I don't see ML mentioned in either IBM Style or the Supplementary Style Guide, and I don't see the combination of "AI/ML" approved.

File: hardware_accelerators/about-hardware-accelerators.adoc (hunk 7 above)
Comment (lahinson) [reply hardware_accelerators/about-hardware-accelerators.adoc:11]: +1 to this comment from Max. Also, do you mean Red Hat OpenShift, or Red Hat OpenShift Container Platform?

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+
+* Graphical processing units (GPUs)
+* Neural processing units (NPUs)
+* Application-specific integrated circuits (ASICs)
+* Data processing units (DPUs)
+* Others
+
+image::OCP_HW_Accelerators_4.png[Supported hardware accelerators cards and peripherals]
+
+Specialized hardware accelerators provide a rich set of benefits for AI/ML development:
```
Comment (lahinson) [inline hardware_accelerators/about-hardware-accelerators.adoc:21]: See my earlier comment about "AI/ML".

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 14]
```diff
@@ -0,0 +1,53 @@
...
+[id="about-hardware-accelerators"]
+= About hardware accelerators
+include::_attributes/common-attributes.adoc[]
+:context: about-hardware-accelerators
+
+toc::[]
+
+Specialized hardware accelerators play a key role in the emerging generative artificial intelligence and machine learning (AI/ML) industry. Specifically, hardware accelerators are essential to the training and serving of large language and other foundational models that power this new technology. It allows data scientists, data engineers, ML engineers, and developers to take advantage of the specialized hardware acceleration for data-intensive transformations and model development and serving. Much of that ecosystem is open source with a number of contributing partners and open source foundations.
+
+Red{nbsp}Hat OpenShift provides support for cards and peripheral hardware that add processing units that comprise hardware accelerators:
```
Comment (lahinson) [inline hardware_accelerators/about-hardware-accelerators.adoc:11]:
```suggestion
Red{nbsp}Hat OpenShift supports cards and peripheral hardware that add processing units that comprise hardware accelerators:
```

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 15]
```diff
@@ -0,0 +1,53 @@
...
+* Neural processing units (NPUs)
+* Application-specific integrated circuits (ASICs)
+* Data processing units (DPUs)
+* Others
+
+image::OCP_HW_Accelerators_4.png[Supported hardware accelerators cards and peripherals]
+
+Specialized hardware accelerators provide a rich set of benefits for AI/ML development:
+
+One platform for all:: Allows for collaborative environments for developers, data engineers, data scientists, and DevOps.
```
Comment (lahinson) [inline hardware_accelerators/about-hardware-accelerators.adoc:23]:
```suggestion
One platform for all:: A collaborative environment for developers, data engineers, data scientists, and DevOps
```
When possible, avoid anthropomorphic language: https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 16]
```diff
@@ -0,0 +1,53 @@
...
+Extended capabilities with Operators:: Operators allow for bringing AI/ML capabilities to OpenShift.
```
Comment (lahinson) [inline hardware_accelerators/about-hardware-accelerators.adoc:24]:
```suggestion
Extended capabilities with Operators:: Operators provide AI and ML capabilities to {product-title}
```

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 17]
```diff
@@ -0,0 +1,53 @@
...
+Hybrid-cloud support:: On-premise support for model development, delivery, and deployment.
```
Comment (lahinson) [inline hardware_accelerators/about-hardware-accelerators.adoc:25]:
```suggestion
Hybrid-cloud support:: On-premise support for model development, delivery, and deployment
```

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 18]
```diff
@@ -0,0 +1,53 @@
...
+* Others
+
+image::OCP_HW_Accelerators_4.png[Supported hardware accelerators cards and peripherals]
+
+Specialized hardware accelerators provide a rich set of benefits for AI/ML development:
+
+One platform for all:: Allows for collaborative environments for developers, data engineers, data scientists, and DevOps.
+Extended capabilities with Operators:: Operators allow for bringing AI/ML capabilities to OpenShift.
+Hybrid-cloud support:: On-premise support for model development, delivery, and deployment. 
+Support for AI/ML workloads:: Model testing, iteration, integration, promotion, and serving into production as services.
```
Comment (lahinson) [inline hardware_accelerators/about-hardware-accelerators.adoc:26]:
```suggestion
Support for AI and ML workloads:: Model testing, iteration, integration, promotion, and serving into production as services
```
This is a little hard to follow. Does the phrase "as services" modify only "serving into production", or does it modify "model testing, iteration, integration, promotion, and serving into production"?

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 19]
```diff
@@ -0,0 +1,53 @@
...
+
+Red{nbsp}Hat provides an optimized platform to enable these specialized hardware accelerators in our RHEL and OpenShift platforms at the Linux (kernel and userspace) and Kubernetes layers. To do this, Red{nbsp}Hat combines the proven capabilities of Red{nbsp}Hat OpenShift AI and Red{nbsp}Hat OpenShift in a single enterprise-ready AI application platform.
```
Comment (lahinson) [inline hardware_accelerators/about-hardware-accelerators.adoc:28]:
```suggestion
Red{nbsp}Hat provides an optimized platform to enable these specialized hardware accelerators in {op-system-base-full} and {product-title} platforms at the Linux (kernel and userspace) and Kubernetes layers. To do this, Red{nbsp}Hat combines the proven capabilities of Red{nbsp}Hat OpenShift AI and {product-title} in a single enterprise-ready AI application platform. 
```

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 20]
```diff
@@ -0,0 +1,53 @@
...
+Extended capabilities with Operators:: Operators allow for bringing AI/ML capabilities to OpenShift.
+Hybrid-cloud support:: On-premise support for model development, delivery, and deployment. 
+Support for AI/ML workloads:: Model testing, iteration, integration, promotion, and serving into production as services.
+
+Red{nbsp}Hat provides an optimized platform to enable these specialized hardware accelerators in our RHEL and OpenShift platforms at the Linux (kernel and userspace) and Kubernetes layers. To do this, Red{nbsp}Hat combines the proven capabilities of Red{nbsp}Hat OpenShift AI and Red{nbsp}Hat OpenShift in a single enterprise-ready AI application platform. 
+
+Hardware Operators use the operating framework of a Kubernetes cluster to enable the required accelerator resources. A device plugin is also provided that can be deployed manually or as a DaemonSet. This plugin enables the registration of the GPU in the cluster. 
+
+Certain specialized hardware accelerators are designed to work within air gapped (disconnected) environments. This is because of customer requirements to maintain a secure environment for development and testing.
```
Comment (lahinson) [inline hardware_accelerators/about-hardware-accelerators.adoc:32]:
```suggestion
Certain specialized hardware accelerators are designed to work within air gapped (disconnected) environments, where a secure environment must be maintained for development and testing.  
```
Trying to avoid starting a sentence with an ambiguous pronoun (This). See https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns#ambiguous-pronoun-references

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 21]
```diff
@@ -0,0 +1,53 @@
...
+Certain specialized hardware accelerators are designed to work within air gapped (disconnected) environments. This is because of customer requirements to maintain a secure environment for development and testing.  
+
+[id="hardware-accelerators"]
+== Hardware accelerators
+
+Red{nbsp}Hat Openshift enables the following hardware accelerators:
+
+* NVIDIA GPU
+* AMD Instinct GPU
+* Intel Gaudi
```
Comment (lahinson) [inline hardware_accelerators/about-hardware-accelerators.adoc:41]:
```suggestion
* Intel(R) Gaudi(R)
```
Mark registered trademark symbols on their first instance.

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 22]
```diff
@@ -0,0 +1,53 @@
...
+.Additional resources
+
+* link:https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2-latest/html/introduction_to_red_hat_openshift_ai/index[Introduction to Red Hat OpenShift AI]
+
+* link:https://docs.nvidia.com/datacenter/cloud-native/openshift/latest/index.html[
+NVIDIA GPU Operator on Red Hat OpenShift Container Platform]
+
+* link:https://www.amd.com/en/products/accelerators/instinct.html[AMD Instinct Accelerators]
+
+* link:https://www.intel.com/content/www/us/en/products/details/processors/ai-accelerators/gaudi-overview.html[Intel® Gaudi® Al Accelerators]
```
Comment (lahinson) [inline hardware_accelerators/about-hardware-accelerators.adoc:53]:
```suggestion
* link:https://www.intel.com/content/www/us/en/products/details/processors/ai-accelerators/gaudi-overview.html[Intel Gaudi Al Accelerators]
```

**Review by lahinson — COMMENTED** [review]
Nice work! Just a few comments for your consideration.

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 23]
```diff
@@ -0,0 +1,53 @@
...
+Specialized hardware accelerators provide a rich set of benefits for AI/ML development:
+
+One platform for all:: A collaborative environment for developers, data engineers, data scientists, and DevOps
+Extended capabilities with Operators:: Operators allow for bringing AI/ML capabilities to OpenShift
+Hybrid-cloud support:: On-premise support for model development, delivery, and deployment 
+Support for AI/ML workloads:: Model testing, iteration, integration, promotion, and serving into production as services
+
+Red{nbsp}Hat provides an optimized platform to enable these specialized hardware accelerators in {op-system-base-full} and OpenShift platforms at the Linux (kernel and userspace) and Kubernetes layers. To do this, Red{nbsp}Hat combines the proven capabilities of Red{nbsp}Hat OpenShift AI and Red{nbsp}Hat OpenShift in a single enterprise-ready AI application platform. 
+
+Hardware Operators use the operating framework of a Kubernetes cluster to enable the required accelerator resources. A device plugin is also provided that can be deployed manually or as a DaemonSet. This plugin enables the registration of the GPU in the cluster.
```
Comment (bergerhoffer) [inline hardware_accelerators/about-hardware-accelerators.adoc:30]: Suggesting for better readability / getting rid of passive voice. Also a suggestion in the second sentence, but not sure if that changes the meaning

```suggestion
Hardware Operators use the operating framework of a Kubernetes cluster to enable the required accelerator resources. You can also deploy the provided device plugin manually or as a daemon set. This plugin registers the GPU in the cluster. 
```

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 24]
```diff
@@ -0,0 +1,53 @@
...
+Extended capabilities with Operators:: Operators allow for bringing AI/ML capabilities to OpenShift
+Hybrid-cloud support:: On-premise support for model development, delivery, and deployment 
+Support for AI/ML workloads:: Model testing, iteration, integration, promotion, and serving into production as services
+
+Red{nbsp}Hat provides an optimized platform to enable these specialized hardware accelerators in {op-system-base-full} and OpenShift platforms at the Linux (kernel and userspace) and Kubernetes layers. To do this, Red{nbsp}Hat combines the proven capabilities of Red{nbsp}Hat OpenShift AI and Red{nbsp}Hat OpenShift in a single enterprise-ready AI application platform. 
+
+Hardware Operators use the operating framework of a Kubernetes cluster to enable the required accelerator resources. A device plugin is also provided that can be deployed manually or as a DaemonSet. This plugin enables the registration of the GPU in the cluster. 
+
+Certain specialized hardware accelerators are designed to work within air gapped (disconnected) environments where a secure environment must be maintained for development and testing.
```
Comment (bergerhoffer) [inline hardware_accelerators/about-hardware-accelerators.adoc:32]: We prefer "disconnected" in OCP, so just that should be fine

https://redhat-documentation.github.io/supplementary-style-guide/#disconnected

```suggestion
Certain specialized hardware accelerators are designed to work within disconnected environments where a secure environment must be maintained for development and testing.  
```

File: hardware_accelerators/about-hardware-accelerators.adoc [hunk 25]
```diff
@@ -0,0 +1,53 @@
...
+Red{nbsp}Hat provides an optimized platform to enable these specialized hardware accelerators in {op-system-base-full} and OpenShift platforms at the Linux (kernel and userspace) and Kubernetes layers. To do this, Red{nbsp}Hat combines the proven capabilities of Red{nbsp}Hat OpenShift AI and Red{nbsp}Hat OpenShift in a single enterprise-ready AI application platform. 
+
+Hardware Operators use the operating framework of a Kubernetes cluster to enable the required accelerator resources. A device plugin is also provided that can be deployed manually or as a DaemonSet. This plugin enables the registration of the GPU in the cluster. 
+
+Certain specialized hardware accelerators are designed to work within air gapped (disconnected) environments where a secure environment must be maintained for development and testing.  
+
+[id="hardware-accelerators"]
+== Hardware accelerators
+
+Red{nbsp}Hat Openshift enables the following hardware accelerators:
```
Comment (bergerhoffer) [inline hardware_accelerators/about-hardware-accelerators.adoc:37]: Same about using {product-title}

**Review by bergerhoffer — COMMENTED** [review]
A few questions/things to address before merging this. Most importantly that this is adding a new book and I'm not sure if that was the intention, since I don't see a note about it in the PR description.

**Review by adellape — APPROVED** [review]
